sound/soc/sof/pm.c

```diff
@@ -94,6 +95,10 @@ static int sof_restore_pipelines(struct snd_sof_dev *sdev)
 						 comp_dai, sizeof(*comp_dai),
 						 &r, sizeof(r));
 			break;
+		case snd_soc_dapm_scheduler:
+			pipeline = (struct sof_ipc_pipe_new *)swidget->private;
+			ret = sof_load_pipeline_ipc(sdev, pipeline, &r);
+			break;
```

Comment:
the commit message is far from clear. What does "power up the pipeline schedule core" mean?
Also you are only doing something on resume, what happens on suspend? It's helpful to have the two parts in the same patch to track the flows, or have a comment that tells explicitly that this not needed for suspend.